lib/fprotect/Kconfig

```diff
@@ -51,6 +51,8 @@ menuconfig FPROTECT
 	depends on SOC_FAMILY_NORDIC_NRF
 	depends on !(SOC_SERIES_NRF54LX && IS_SECURE_BOOTLOADER)
 	select NRFX_RRAMC if SOC_SERIES_NRF54LX
+	default y if MCUBOOT && (!SOC_SERIES_NRF54LX || (SOC_SERIES_NRF54LX && MCUBOOT_MCUBOOT_IMAGE_NUMBER = -1))
```

Comment:
please add comment there about reasoning, something like: enabled by default for MCUboot build as only bootloader on nRF54L SoC's. Similar below, mcuboot might use both RRAM regions 3 & 4 as it's allone.